File: 30DaysOfCode/day25RunningTimeComplexity.c
#include <stdio.h>
#include <string.h>
#include <math.h>
#include <stdlib.h>

void checkPrime(int number) {
    printf("%d\n", number);
}

int main() {

    /* Enter your code here. Read input from STDIN. Print output to STDOUT */  
    int T;
    scanf("%d", &T);
    while (T-- > 0) {
        int i;
        scanf("%d", &i);
        checkPrime(i);
    }
    return 0;
}

File: 30DaysOfCode/day25RunningTimeComplexity.py
import math

def checkPrime(number):
    if (number == 1):
        print("Not prime")
        return
    for i in range(2, int(math.sqrt(number) + 1)):
        if (number % i == 0):
            print "Not prime"
            return
    print "Prime"
    
def Solution():
    T = int(raw_input());
    for i in range(T):
        checkPrime(int(raw_input()))
        
Solution()